README.md

```diff
@@ -1,100 +1,68 @@
-
-## Building from Source
-
-### Dependencies
-Before building, you will need:
-- GCC >= 9
-- X11 development files
-- glslang
-- SPIR-V Headers
-- Vulkan Headers
-
-### Building
-
-**These instructions use `--prefix=/usr`, which is generally not recommened since vkBasalt will be installed in directories that are meant for the package manager. The alternative is not setting the prefix, it will then be installed in `/usr/local`. But you need to make sure that `ld` finds the library since /usr/local is very likely not in the default path.** 
-
-In general, prefer using distro provided packages.
+It is also possible to use ReShade FX shaders.
 
+## Downloading
+### [Debian](https://tracker.debian.org/pkg/vkbasalt)
 ```
-git clone https://github.com/DadSchoorse/vkBasalt.git
-cd vkBasalt
+sudo apt install vkbasalt
 ```
 
-#### 64bit
-
+### [Fedora](https://src.fedoraproject.org/rpms/vkBasalt)
 ```
-meson --buildtype=release --prefix=/usr builddir
-ninja -C builddir install
+sudo dnf install vkBasalt
 ```
-#### 32bit
 
-Make sure that `PKG_CONFIG_PATH=/usr/lib32/pkgconfig` and `--libdir=lib32` are correct for your distro and change them if needed. On Debian based distros you need to replace `lib32` with `lib/i386-linux-gnu`, for example.
+### [Void Linux](https://github.com/void-linux/void-packages/blob/master/srcpkgs/vkBasalt/template)
 ```
-ASFLAGS=--32 CFLAGS=-m32 CXXFLAGS=-m32 PKG_CONFIG_PATH=/usr/lib32/pkgconfig meson --prefix=/usr --buildtype=release --libdir=lib32 -Dwith_json=false builddir.32
-ninja -C builddir.32 install
+sudo xbps-install vkBasalt
 ```
 
-## Packaging status
-
-[Debian](https://tracker.debian.org/pkg/vkbasalt) `sudo apt install vkbasalt`
-
-[Fedora](https://src.fedoraproject.org/rpms/vkBasalt) `sudo dnf install vkBasalt`
-
-[Void Linux](https://github.com/void-linux/void-packages/blob/master/srcpkgs/vkBasalt/template) `sudo xbps-install vkBasalt`
-
 ## Usage
 Enable the layer with the environment variable.
 
+### Lutris
+Follow these steps with Lutris:
+1. Right-click on a game and click on `Configure`.
+2. Go to the `System options` tab and scroll down to `Environment variables`.
+3. Click on `Add` and add `ENABLE_VKBASALT` under `Key` and `1` under `Value`.
+
 ### Standard
-When using the terminal or an application (.desktop) file, execute:
+When using the terminal or an application (.desktop) file, run:
 ```ini
 ENABLE_VKBASALT=1 yourgame
 ```
 
-### Lutris
-With Lutris, follow these steps below:
-1. Right click on a game, and press `configure`.
-2. Go to the `System options` tab and scroll down to `Environment variables`.
-3. Press on `Add`, and add `ENABLE_VKBASALT` under `Key`, and add `1` under `Value`.
-
 ### Steam
-With Steam, edit your launch options and add:
+Edit your launch options with Steam and add:
 ```ini
 ENABLE_VKBASALT=1 %command% 
 ```
 
-## Configure
+## Settings
+Settings like the CAS sharpness amount can be changed in the config file.
```

Comment:
IMO, commas made it easier for me to read. I think it would help others too.
```suggestion
Settings, like the CAS sharpness amount, can be changed in the config file.
```